### PR TITLE
(docs) update appendix.rst

### PR DESCRIPTION
**Pull request by vladdoster**
## Description of the Pull Request (PR):

Corrects the following:
- grammar
- spelling
- split infinitives
- punctuation

## This fixes or addresses the following GitHub issues:

N/A

### CODE REVIEW

File: appendix.rst [hunk 1]
```diff
@@ -279,9 +279,8 @@ The Library keyword is optional. It will default to
 
     Fingerprints: 22045C8C0B1004D058DE4BEDA20C27EE7FF7BA84
 
-The Fingerprints keyword is optional. It specifies one or more comma
-separated fingerprints corresponding to PGP public keys. If present,
-the bootstrap image will be verified and the build will only proceed
+The Fingerprints keyword is optional. It specifies one or more comma-separated fingerprints corresponding to PGP public keys. If present,
```
Comment (dtrudg) [inline appendix.rst:282]: If the existing text is hard wrapped please ensure changes are hard-wrapped, and are not long lines.

File: appendix.rst [hunk 2]
```diff
@@ -643,20 +642,19 @@ downloading the OS.
 
 The Include keyword is optional. It allows you to install additional packages
 into the core operating system. It is a best practice to supply only the bare
-essentials such that the ``%post`` section has what it needs to properly
-complete the build.
+essentials such that the ``%post`` section has what it needs to complete the build properly.
```
Comment (dtrudg) [inline appendix.rst:645]: If the existing text is hard wrapped please ensure changes are hard-wrapped, and are not long lines.

File: appendix.rst [hunk 3]
```diff
@@ -873,11 +871,10 @@ Note that differently from the ``docker://`` bootstrap agent both ``docker-daemo
 ``scratch`` bootstrap agent
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The scratch bootstrap agent allows you to start from a completely
-empty container. You are then responsible for adding any and all
-executables, libraries etc. that are required. Starting with a scratch
-container can be useful when you are aiming to minimize container size,
-and have a simple application / static binaries.
+The scratch bootstrap agent allows you to start from an empty container. You are then responsible for adding any
```
Comment (dtrudg) [inline appendix.rst:874]: The `completely empty` and `any and all` in the original text here are valid to emphasize the nature of a scratch container and do not need to be removed.

**Review by dtrudg — CHANGES_REQUESTED** [review]
Thank you for the contribution! I agree with most of the changes here, but not all, and if text was previously wrapped <80 columns this should be maintained.

**Comment by brianclemens** [comment]
Closed for inactivity.

**Pull request closed without merging**